grin/src/AbstractInterpretation/Reduce.hs

```diff
@@ -204,12 +205,20 @@ evalInstruction = \case
       let mergedFields = mconcat . (map Data.Foldable.fold) . Map.elems $ tagMap
       selectReg dstReg.simpleType %= (mappend mergedFields)
 
+    -- TODO: review this
+    EveryNthField n -> do
+      tagMap <- use $ selectTagMap srcReg
+      let mergedNthFields = mconcat . mapMaybe (V.!? n) . Map.elems $ tagMap
+      selectReg dstReg.simpleType %= (mappend mergedNthFields)
+
   Extend {..} -> do
     -- TODO: support all selectors
     value <- use $ selectReg srcReg.simpleType
     case dstSelector of
       NodeItem tag itemIndex -> selectTagMap dstReg.at tag.non mempty.ix itemIndex %= (mappend value)
       AllFields -> selectTagMap dstReg %= (Map.map (V.map (mappend value)))
+      -- TODO: review this
```

Comment:
Please could you comment on this? As a reader it is not clear how to and what to review?

Reply:
These are the new instructions added to the abstract machine. They are used in LVA at the default case. They set all the "tags" live. Tags are the very first fields of a node in this representation. I did not find any other instructions that could express this, that's why I added a new instruction.

Reply:
Got it, thanks.